### PR TITLE
Create a new getting started guide

### PR DESCRIPTION
**Pull request by brianquinlan**
The intention is that this should provide complete getting started documentation. Please let me know if I should add something.

### CODE REVIEW

File: GETTING-STARTED.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+## Setup and Installation
+
+### Prerequisites
+
+In order to use **Django Deploy**, you must first install the following dependencies:
+- [Python](https://www.python.org/downloads/) 3.5 or higher
+- [virtualenv](https://virtualenv.pypa.io/en/stable/installation/)
+- [Docker](https://docs.docker.com/install/overview/) (any edition)
+- [Google Cloud SDK](https://cloud.google.com/sdk/docs/quickstarts)
+- [kubectl](https://cloud.google.com/kubernetes-engine/docs/quickstart) (can be
```
Comment (wangxf123456) [inline GETTING-STARTED.md]: kubectl is not needed anymore.

File: GETTING-STARTED.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
+# Getting Started
+
+## Setup and Installation
+
+### Prerequisites
+
+In order to use **Django Deploy**, you must first install the following dependencies:
+- [Python](https://www.python.org/downloads/) 3.5 or higher
```
Comment (enriquejosepadilla) [inline GETTING-STARTED.md]: Should we tell users to check with python --version ?

File: GETTING-STARTED.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+[Setup and Installation steps](#Setup-and-Installation) and running this
+command:
+```bash
+$ django-cloud-deploy new
+```
+
+Follow the prompts displayed in the terminal. Make sure that you remember the
+following information:
+
+ - the default database password
```
Comment (enriquejosepadilla) [inline GETTING-STARTED.md]: Is the password a default?

File: GETTING-STARTED.md (hunk 1 above)
Comment (brianquinlan) [reply GETTING-STARTED.md]: Removed.

File: GETTING-STARTED.md (hunk 2 above)
Comment (brianquinlan) [reply GETTING-STARTED.md]: I added a note in the installation section but I'm not super happy with how I did it...

File: GETTING-STARTED.md (hunk 3 above)
Comment (brianquinlan) [reply GETTING-STARTED.md]: Update the phrasing.

File: GETTING-STARTED.md (hunk 2 above)
Comment (enriquejosepadilla) [reply GETTING-STARTED.md]: i think its clear on our github about 3.5, at this point if that fails for them then GG, else we can add another check to make sure their python and fail fasT?

**Pull request merged**